docs/concepts/validation_decorator.md

```diff
@@ -219,12 +219,12 @@ except ValidationError as e:
 
 
 @validate_call
-def when(dt: datetime = Field(default_factory=datetime.now)):
-    return dt
+def return_value(value: str = Field(default='default value')):
```

Comment:
Simplified the example, as it was a bit confusing to have `type(when())` checked